sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -58,15 +55,21 @@
         <bpmn:flowNodeRef>Event_1851d60</bpmn:flowNodeRef>
         <bpmn:flowNodeRef>Event_1y105g5</bpmn:flowNodeRef>
         <bpmn:flowNodeRef>Event_076thnd</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Event_09fw0dz</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Event_123oime</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Event_0s8ztj5</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Event_18110sl</bpmn:flowNodeRef>
         <bpmn:flowNodeRef>Gateway_0w90zd7</bpmn:flowNodeRef>
+        <bpmn:flowNodeRef>Event_0r8wyou</bpmn:flowNodeRef>
```

Comment:
Can we follow the same pattern from the other on the right?
![image](https://github.com/bcgov/SIMS/assets/61259237/05ae99dc-79da-4222-873e-905895705b98)